packages/core/src/ceramic.ts

```diff
@@ -422,12 +425,14 @@ class Ceramic implements CeramicApi {
     const ipfsTopology = new IpfsTopology(ipfs, networkOptions.name, logger)
     const pinStoreFactory = new PinStoreFactory(ipfs, pinStoreOptions)
     const repository = new Repository(streamCacheLimit, concurrentRequestsLimit, logger)
+    const recordCache = new LRUMap<CID, any>(IPFS_CACHE_SIZE)
```

Comment:
nitpick, but fwiw I think it'd be fine for the record cache to be considered an implementation detail of the Dispatcher, and thus constructed within it rather than passed in.  Not a big deal either way though